homeassistant/components/lifx/manager.py

```diff
@@ -375,6 +409,39 @@ async def start_effect(
             )
             await self.effects_conductor.start(effect, bulbs)
 
+        elif service == SERVICE_EFFECT_SKY:
+            palette = kwargs.get(ATTR_PALETTE, None)
+            if palette is not None:
+                theme = Theme()
+                for hsbk in palette:
+                    theme.add_hsbk(hsbk[0], hsbk[1], hsbk[2], hsbk[3])
+
+            speed = kwargs.get(ATTR_SPEED, EFFECT_SKY_DEFAULT_SPEED)
+            sky_type = kwargs.get(ATTR_SKY_TYPE, EFFECT_SKY_DEFAULT_SKY_TYPE)
+
+            cloud_saturation_min = kwargs.get(
+                ATTR_CLOUD_SATURATION_MIN,
+                EFFECT_SKY_DEFAULT_CLOUD_SATURATION_MIN,
+            )
+            cloud_saturation_max = kwargs.get(
+                ATTR_CLOUD_SATURATION_MAX,
+                EFFECT_SKY_DEFAULT_CLOUD_SATURATION_MAX,
+            )
+
+            await asyncio.gather(
+                *(
+                    coordinator.async_set_matrix_effect(
```

Comment:
To speed this up, wrap it in `create_eager_task`

Reply:
Does this need to be sped up?

Reply:
not required but will reduce latency

Reply:
There are a bunch of other places where this would be far more effective/noticable, I think. I'm not too worried about how long it takes to trigger a firmware effect in this instance, even across multiple devices.